SearchFile.java

```diff
@@ -1,5 +1,36 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class SearchFile{
-  static void main(String[] args){
-	  System.out.println("Hello World");
-  }
+	
+	public SearchFile() {
+		
+	}
+	
+	public ArrayList<String> search(String pattern, String file) throws FileNotFoundException{
+		Scanner fileScanner = new Scanner(new File(file));
+		ArrayList<String> listOfLines = new ArrayList<String>();
+		while(fileScanner.hasNextLine()) {
+			String line = fileScanner.nextLine();
+			if(line.contains(pattern)) {
+				listOfLines.add(line);
+			}
+		}
+		
+		return listOfLines;
+	}
+	
+	public static void main(String[] args) throws FileNotFoundException{
+		SearchFile sf = new SearchFile();
+		ArrayList<String> list = sf.search("Hej", "cat.txt");
```

Comment:
Borde generaliseras så att input kan tas in via args[] på formatet: "search <pattern> <file>"

Reply:
håller med, fixar